assets/javascript/app.js

```diff
@@ -16,15 +15,15 @@ $(document).ready(function() {
             timeout: 5000,
             maximumAge: 0
         };
-          
+        
+        // If geolocation is accepted set map + nearby brewery search to the user's location
         function success(pos) {
-            var crd = pos.coords;
```

Comment:
Removed because lat + long are pull direct from pos. Setting a variable is redundant.